server/src/shared/infra/db/entities/auth-code.entity.ts

```diff
@@ -8,6 +8,9 @@ export class AuthCodeEntity extends BaseEntity {
   @Property()
   clientId!: string
 
+  @Property()
+  userId!: string
```

Comment:
when the time comes to actually use these entities with a DB (e.g. integration tests), remember to create a migration